lib/Mojolicious/Plugin/OpenAPI.pm

```diff
@@ -257,6 +263,75 @@ sub _route_path {
   return $path;
 }
 
+sub _security_action {
+  my ($self, $api_spec) = @_;
+  my $global      = $api_spec->get('/security') || [];
+  my $definitions = $api_spec->get('/securityDefinitions');
+  my $security_cb = $self->_security_cb;
+
+  return sub {
+    my $c = shift;
+    my @security = @{ $c->openapi->spec->{security} || $global };
```

Comment:
No, it is overridden entirely. This is confirmed by the fact that the documentation says explicitly that a set of global security policies can be ignored by setting the op security to an empty array